primitives/chain-bridge-hub-cumulus/src/lib.rs

```diff
@@ -55,9 +57,9 @@ parameter_types! {
 		NORMAL_DISPATCH_RATIO,
 	);
 
-	pub const BlockExecutionWeight: Weight = constants::WEIGHT_PER_NANOS.saturating_mul(5_000_000);
+	pub const BlockExecutionWeight: Weight = Weight::from_ref_time(constants::WEIGHT_REF_TIME_PER_SECOND).saturating_mul(5_000_000);
```

Comment:
Is this change (`WEIGHT_PER_NANOS * 5_000_000` -> WEIGHT_REF_TIME_PER_SECOND * 5_000_000`) correct? The difference looks too big

Reply:
ou, thank you